api/src/main/java/io/cloudevents/format/BinaryUnmarshaller.java

```diff
@@ -133,9 +133,9 @@ private BinaryUnmarshaller() {}
 		 */
 		HeadersStep<A, T, P> builder(EventBuilder<T, A> builder);
 	}
-	
+
 	private static final class Builder<A extends Attributes, T, P> implements
```

Comment:
this still has the layer cake of generics

Reply:
This still needs to be removed :smile: